src/Commands/Site/ImportCommand.php

```
@@ -0,0 +1,40 @@
+<?php
+namespace Pantheon\Terminus\Commands\Site;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Terminus\Exceptions\TerminusException;
+
+class ImportCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    /**
+     * Imports a site archive onto a Pantheon site
+     *
+     * @authorized
+     *
+     * @name site:import
+     * @alias import
+     *
+     * @option string $site Name of the site to import to
+     * @option string $url  URL at which the import archive exists
+     * @usage terminus import --site=<site_name> --url=<archive_url>
+     *   Imports the file at the archive URL to the site named.
+     */
+    public function import($sitename, $url)
+    {
+        $site = $sitename.'.dev';
+        list(, $env) = $this->getSiteEnv($site);
+        $workflow = $env->import($url);
+        try {
+            $workflow->wait();
+        } catch (\Exception $e) {
+            if ($e->getMessage() == "Successfully queued import_site") {
```

This is a bit brittle, since that message is generated in Ygg and is actually incorrect, so someone might fix it. I think for now, we let the exceptions bubble up, or fix the problem in Ygg.

---

@marktheunissen  I agree this should be fixed in Ygg. 
Ideally this just falls out and keeps working - in any other case it just passes the Ygg message. Also - @TeslaDethray helped me with this one - so looping in.

---

I do think that we should throw here, but I'm torn here - I wanted to check the message because there are other errors which may bubble up and I wanted to ensure that this specific one is corrected. Is there any way we can just go and fix that message once and for all and remove this check, @marktheunissen?